rpcs3/Emu/SysCalls/Modules/cellFont.cpp

```diff
@@ -2,7 +2,10 @@
 #include "Emu/Memory/Memory.h"
 #include "Emu/SysCalls/Modules.h"
 
+// Defines STB_IMAGE_IMPLEMENTATION *once* before including stb_truetype.h (as said in stb_truetype.h's comments)
+#define STB_TRUETYPE_IMPLEMENTATION
```

Comment:
Should this be defined here? 
AFAIK this is the only place stb_truetype is used so it should work